stock_picking_backorder_partner_config/views/res_partner_view.xml

```diff
@@ -0,0 +1,15 @@
+<odoo>
+
+    <record id="res_partner_form_view" model="ir.ui.view">
+        <field name="name">res.partner.form.view (stock_picking_backorder_partner_config)</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='category_id']" position="after">
```

Comment:
There's an Inventory section under Sales & Purchase page, this could be a better location for theses settings